modules/rostests/apitests/shlwapi/SHPropertyBag.cpp

```diff
@@ -83,6 +132,36 @@ class CDummyWritePropertyBag : public IPropertyBag
     }
 };
 
+static void SHPropertyBag_ReadTest(void)
```

Comment:
Actually, it would be better if the tests that use the Write* functions also called the Read* functions in order to show that what is being written to the property bag, also gets correctly read, and the data being written is the same as the data being read afterwards.

Reply:
Hereby, `IPropertyBag` is virtualized by dummy classes.
This testcase just sees the read/write triggers of `*Read*` and `*Write*` functions.
Actual read/write operations against the trigger are beyond the purpose of this testcase.

Reply:
It would be nice to have a new testcase that can do the write + read back what has been written to verify that everything has been done correctly.

Reply:
United to `CDummyPropertyBag`.